west.yml

```diff
@@ -62,7 +62,7 @@ manifest:
       groups:
         - hal
     - name: hal_espressif
-      revision: 2f438dfde14900d35bf703ddc4c0f71efc0d065f
+      revision: c41ca19291a8d4cfcf57c52a1ca3b0262e5b2cf7
```

Comment:
This doesn't seem to be needed. It changes revision within hal_espressif, but actually none of the code is changed. It looks like required changes were silently pulled by #38955.

Please note that removing `feature/esp32c3_wifi` (that points to 2f438dfde14900d35bf703ddc4c0f71efc0d065f) will result in breaking bisectability of Zephyr!

Reply:
@mniestroj sure. Updated commit message and kept revision. Thanks.

Reply:
Sorry, I re-requested review. thanks for pointing that out.

I was curious to know if this change is necessary as well. On a similar note, the release team would like to begin enabling CI for esp32 soon, and we would very much like to see toolchain and hal maturing. Is there anything we can do do help?

Reply:
Is it possible to use Zephyr's CI with custom espressif toolchain?

Reply:
I have a fully `twister -p esp32` working in here using a new toolchain and some linker fixes that would allow that, but requires Zephyr's CI to use Espressif's toolchain for that.

Reply:
> This doesn't seem to be needed. It changes revision within hal_espressif, but actually none of the code is changed. It looks like required changes were silently pulled by #38955.
> 
> Please note that removing `feature/esp32c3_wifi` (that points to 2f438dfde14900d35bf703ddc4c0f71efc0d065f) will result in breaking bisectability of Zephyr!

This revision (2f438dfde14900d35bf703ddc4c0f71efc0d065f) comes from a PR in hal_espressif. It should had been from `hal_espressif` HEAD instead. That's the reason I changed it. It was the same commit though.

Reply:
@sylvioalves - I would say the best thing to do is create an issue for CI / toolchain integration, with some proposals, add the `dev-review` label, then plan on joining the next [Zephyr Dev Meeting](https://github.com/zephyrproject-rtos/zephyr/wiki/Zephyr-Committee-and-Working-Group-Meetings#zephyr-dev-meeting).

Reply:
> @sylvioalves - I would say the best thing to do is create an issue for CI / toolchain integration, with some proposals, add the `dev-review` label, then plan on joining the next [Zephyr Dev Meeting](https://github.com/zephyrproject-rtos/zephyr/wiki/Zephyr-Committee-and-Working-Group-Meetings#zephyr-dev-meeting).

Sounds like a plan. I'll do that. Thanks.